src/server/scripts/Northrend/IcecrownCitadel/boss_sindragosa.cpp

```diff
@@ -1297,46 +1298,42 @@ class spell_sindragosa_ice_tomb : public SpellScriptLoader
                 return true;
             }
 
-            void SummonTomb()
+            void PeriodicTick(AuraEffect const* aurEff)
```

Comment:
Add a check in this method to only summon the block once the player stops moving. similar to the Sapphiron changes in #16524.